examples/phi2/phi2_slicegpt.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
Instead of adding new .json file, could you just leverage template .json and update phi2.py to add "slicegpt" pass based on a command line argument?

Reply:
Updated.